fdb-record-layer-lucene/src/main/java/org/apache/lucene/codecs/lucene84/package-info.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * package-info.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Support for {@code LUCENE} indexes and queries.
```

Comment:
This looks like it's been taken from the `com.apple.foundationdb.record.lucene` package-info. This should probably describe the code in this subpackage specifically, including noting that the codec is adapted from the 8.11 optimized codec